### ISSUE TITLE
[Bug]: 

### ISSUE THREAD

**Issue by fra1993** [speechbrain/speechbrain#1927]
### Describe the bug

When trying to run the CommonLanguage recipe in multigpu, I get the following warning: `UserWarning: Grad strides do not match bucket view strides. This may indicate grad was not created according to the gradient layout contract, or that the param's strides changed since DDP was constructed.  This is not an error, but may impair performance.
grad.sizes() = [192, 6144, 1], strides() = [6144, 1, 6144]
bucket_view.sizes() = [192, 6144, 1], strides() = [6144, 1, 1] (Triggered internally at ../torch/csrc/distributed/c10d/reducer.cpp:325.)
  Variable._execution_engine.run_backward(  # Calls into the C++ engine to run the backward pass
`

I found [some suggestions](https://github.com/lucidrains/DALLE2-pytorch/issues/196) of adding `.contiguous()` when changing the dimensions (like transposing) in the ECAPA part of lang_ID but this didn't help.

### Expected behaviour

No warning.

### To Reproduce

python -m torch.distributed.launch --nproc_per_node=4 train.py $YAML --distributed_launch --distributed_backend='nccl'
